README.md

```diff
@@ -1,17 +1,25 @@
 <p align="center">
-  <img src="https://github.com/mindee/doctr/releases/download/v0.3.1/Logo_doctr.gif?raw=True" width="40%">
+  <img src="docs/images/Logo_doctr.gif" width="40%">
 </p>
 
 [![Slack Icon](https://img.shields.io/badge/Slack-Community-4A154B?style=flat-square&logo=slack&logoColor=white)](https://slack.mindee.com) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) ![Build Status](https://github.com/mindee/doctr/workflows/builds/badge.svg) [![codecov](https://codecov.io/gh/mindee/doctr/branch/main/graph/badge.svg?token=577MO567NM)](https://codecov.io/gh/mindee/doctr) [![CodeFactor](https://www.codefactor.io/repository/github/mindee/doctr/badge?s=bae07db86bb079ce9d6542315b8c6e70fa708a7e)](https://www.codefactor.io/repository/github/mindee/doctr) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/340a76749b634586a498e1c0ab998f08)](https://app.codacy.com/gh/mindee/doctr?utm_source=github.com&utm_medium=referral&utm_content=mindee/doctr&utm_campaign=Badge_Grade) [![Doc Status](https://github.com/mindee/doctr/workflows/doc-status/badge.svg)](https://mindee.github.io/doctr) [![Pypi](https://img.shields.io/badge/pypi-v0.7.0-blue.svg)](https://pypi.org/project/python-doctr/) [![Hugging Face Spaces](https://img.shields.io/badge/%F0%9F%A4%97%20Hugging%20Face-Spaces-blue)](https://huggingface.co/spaces/mindee/doctr) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mindee/notebooks/blob/main/doctr/quicktour.ipynb)
 
+
```

Comment:
don't think the pypi badge and installation guide inside the docs are enough ? :)

Reply:
@odulcy-mindee https://github.com/mindee/doctr#installation

Reply:
@felixT2K We can add a link to this header instead, wdyt ?

Reply:
That would be fine but than moving it as first point after the Quicktour headline ?

Reply:
@felixT2K Well, in the end, I reverted that. I'm eager to update the README.md but let's do that in another PR.